tests/base/boring_model.py

```diff
@@ -55,8 +56,9 @@ def __len__(self):
 
 
 class BoringModel(LightningModule):
-
-    def __init__(self):
+    def __init__(self, in_features: int = 32, out_features: int = 2,
+                 optimizer_name: str = 'SGD', learning_rate: float = 0.1,
+                 batch_size:int = 1):
```

Comment:
I'd still suggest not adding any arguments or any other assignments in BoringModel. Recently I added `example_input_array` which is a lightning thing but William suggested to remove that and instead override Boring Model to keep BoringModel as it is.

Reply:
Thank you @rohitgr7! @Borda can you help me on this? What do you suggest?